StratisCore.UI/src/app/wallet/staking/staking.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container>
-  <div *ngIf="!wallet || wallet.hasBalance">
```

Comment:
Because the GetHistory api call is relatively slow we can have a situation where we have a Balance and no Transactions.
The ticket bug is to hide staking window when we don't have any transactions available.

Suggest that fix is done on the Dashboard Component html line 50
[https://github.com/stratisproject/StratisCore/blob/master/StratisCore.UI/src/app/wallet/dashboard/dashboard.component.html#L50](https://github.com/stratisproject/StratisCore/blob/master/StratisCore.UI/src/app/wallet/dashboard/dashboard.component.html#L50)
```<div class="col-4" *ngIf="(transactionCount | async) > 0 && !globalService.getSidechainEnabled() && !walletService.ibdMode">
        <app-staking></app-staking>
      </div>
```

Reply:
Weird. This one got overwritten somehow. I added it back